sample-code/examples/python/ios_complex.py

```diff
@@ -24,10 +24,7 @@ def setUp(self):
         # set up appium
         # ** Important Note **
         # Make sure you have build the UICatalog applcation in your local repository
-        app = os.path.join(os.path.dirname(__file__),
-                           '../../apps/UICatalog/build/release-iphonesimulator',
-                           'UICatalog.app')
-        app = os.path.abspath(app)
+        app = os.path.abspath('../../apps/UICatalog/build/release-iphonesimulator/UICatalog.app')
```

Comment:
this means that the path will be different based on the working directory. the existing strategy is to make it absolute, based on the location of this file. I think we probably still need it the way it is. You could join the last two lines, though.